client/my-sites/upgrades/cart/test/cart-buttons.js

```diff
@@ -1,65 +1,68 @@
 /**
  * External Dependencies
  */
-const sinon = require( 'sinon' ),
-	ReactDom = require( 'react-dom' ),
-	React = require( 'react' ),
-	TestUtils = require( 'react-addons-test-utils' ),
-	{ expect } = require( 'chai' );
+import sinon from 'sinon';
+import React from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
```

Comment:
Perhaps not possible in this case, but if at all possible, we should try to use `shallow` and avoid a fake DOM. Did you test to see whether this was possible to do here?